tests/ui.rs

```diff
@@ -18,5 +18,24 @@ fn main() -> color_eyre::eyre::Result<()> {
     // Make sure we can depend on duchess itself in our tests
     config.dependencies_crate_manifest_path = Some("Cargo.toml".into());
 
-    run_tests(config)
+    let test_name = std::env::var_os("TESTNAME");
```

Comment:
Useful, though using an env var feels vaguely annoying. I feel like this should be built into the framework.

Reply:
It could also be `let test_name = env::args().nth(1);`, to enable `cargo test test-name`.